R/article.R

```diff
@@ -309,6 +309,25 @@ sage_article <- function(..., highlight = NULL, citation_package = "natbib") {
   )
 }
 
+#' @section \code{science_article}: Format for AAAS - Science articles
+#' Based on the official Science Journal LaTeX guidelines
+#' http://www.sciencemag.org/authors/preparing-manuscripts-using-latex 
+#' Most article formats are based on \code{rmarkdown::pdf_document()}, with a
+#' custom Pandoc LaTeX template and different default values for other arguments
+#' (e.g., \code{keep_tex = TRUE}).
+#'
+#' @name science_article
+#' @export
+#' @rdname article
+science_article <- function(
+  ..., keep_tex = TRUE, md_extensions = c("-autolink_bare_uris")
+) {
+  pdf_document_format(
+    "science", md_extensions = md_extensions, ...
```

Comment:
`keep_tex` is defined above but is missing
```suggestion
    "science", keep_tex = keep_tex, md_extensions = md_extensions, ...
```